internal/environment/environment.go

```diff
@@ -54,17 +54,22 @@ func NewPaths(base string) Paths {
 // BasePath returns krew base directory.
 func (p Paths) BasePath() string { return p.base }
 
+// IndexBase returns the krew index directory.
+func (p Paths) IndexBase() string {
```

Comment:
I think this is a good addition. Most callers of `IndexPath` will in the future call this function instead. I think it would make sense to switch all the other calling sites which look like `IndexPath("")` to this function.

Could you also mention in the godoc that the base includes all the indexes (default + custom ones).

Reply:
Ya I was thinking about that. I think though that most uses after the multi index changes would still use `IndexPath(name)` right?

I'll update the godoc for it.

Reply:
It depends. I think all those usages in connection to `LoadPluginListFromFS` will use the index base. And  there are a few of those :)

Reply:
Wouldn't those still use `IndexPath(name)`? It would need to look in `$KREW_ROOT/index/.../plugins` so I'd think the calls to `LoadPluginListFromFS` would still be passed `IndexPluginsPath(name)`.

Let me know if I've misunderstood something.

Reply:
Yeah, that's right. But we need to discover the indices by listing the index base directory.

Anyway, I don't think it matters too much right now. We have to revisit later anyway.